### DISCUSSION TITLE
Kubernetes Compatibility Inquiry for Pelican

### DISCUSSION THREAD

**Discussion by ClemCreator** [pelican-dev/panel#513] [General]
Hello Pelican Team,

Firstly, thank you for developing and maintaining Pelican, an outstanding game server control panel that offers high security and a user-friendly interface. 

Could you please let me know if Kubernetes compatibility is on your roadmap? If not, are there any specific reasons or challenges preventing this integration? I believe this would be a valuable addition for some users looking to manage large-scale game server deployments.

Thank you for your time and effort in continuously improving Pelican

**Comment by rmartinoscar** [comment]
![image](https://github.com/user-attachments/assets/1ce43a1f-c8b7-4354-bc51-9fb81ff473c8)
